tests/system/action/test_internal_actions.py

```diff
@@ -88,22 +74,3 @@ def test_internal_no_password_in_request(self) -> None:
         response = self.internal_request("user.create", {"username": "test"}, None)
         self.assert_status_code(response, 401)
         self.assert_model_not_exists("user/2")
-
-    @patch("openslides_backend.shared.env.is_dev_mode")
-    def test_internal_no_password_on_server(self, is_dev_mode: Mock) -> None:
-        is_dev_mode.return_value = False
-        del os.environ[INTERNAL_AUTH_PASSWORD_FILE]
-        response = self.internal_request("user.create", {"username": "test"})
-        self.assert_status_code(response, 500)
-        self.assert_model_not_exists("user/2")
-
-    @patch("openslides_backend.shared.env.is_dev_mode")
-    def test_internal_try_access_backend_internal_action(
-        self, is_dev_mode: Mock
-    ) -> None:
-        is_dev_mode.return_value = False
-        response = self.internal_request(
-            "option.create", {"meeting_id": 1, "text": "test"}
-        )
-        self.assert_status_code(response, 400)
-        self.assert_model_not_exists("option/1")
```

Comment:
If you keep the `is_dev_mode` method, you can also keep these tests. Also, how are they wrong?

Reply:
I reinserted the test but changed some things. The env vars are set only during `setUp` so it is not possible to manipulate them inside a single test. Now everything should work.